src/bci_build/package/basecontainers.py

```diff
@@ -199,6 +200,39 @@ def _get_supported_until_fips(os_version: OsVersion) -> datetime.date:
 ]
 
 
+FIPS_MICRO_CONTAINERS = [
+    OsContainer(
+        name="micro-fips",
+        os_version=os_version,
+        support_level=SupportLevel.L3,
+        supported_until=_get_supported_until_fips(os_version),
+        logo_url="https://opensource.suse.com/bci/SLE_BCI_logomark_green.svg",
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        pretty_name=_get_fips_pretty_name(os_version),
+        from_target_image=generate_from_image_tag(os_version, "bci-micro"),
+        build_recipe_type=BuildType.DOCKER,
+        package_list=[name for name in ["coreutils", "crypto-policies-scripts", *os_version.release_package_names]
```

Comment:
I don't think we should be shipping crypto-policies-scripts, this is a huge dependency. instead we should be just adding the libraries and their hmac files for which we provide them separately.

Reply:
what's the point of the double list comprehension? seems confusing.